src/components/orders/OrderPriceDisplay/index.tsx

```diff
@@ -58,7 +58,10 @@ export function OrderPriceDisplay(props: Props): JSX.Element {
 
   return (
     <Wrapper>
-      {formattedPrice} {quoteSymbol} for {baseSymbol}
+      <span>
+        {formattedPrice} {quoteSymbol}&nbsp;
+      </span>{' '}
```

Comment:
Do you need both `&nbsp;` and `{' '}` ?

Reply:
I wanted to have a space in between, because in the regular order overview the space is missing without...

Reply:
Seems not relevant anymore in latest commit 👍🏼